FrontMen/fm-hours
# fix: send timesheet denial message via email

# Changes

## Changed

Only show the deny button if the status is pending. The button was working before. The issue was that when the timesheet had the status "new," for example, it would do anything.

## How to test

Check if the button only shows for pending timesheets, and check if when clicking on deny, the footer button changes, and you receive a confirmation email.
